steps/commenter/0.0.1/step.yml

```diff
@@ -0,0 +1,67 @@
+title: Commenter
```

Comment:
I would call it `Add GitHub comment` or something like that. `Commenter` is too broad.

Reply:
I agree with @koral-- , we are working on an update for the step development guideline, one of the updates will cover step title requirements. Pls use this layout for title: `<verb/action> <service/tool>`, like `Add GitHub comment`

Reply:
Fixed in `v0.0.2` 👍